luigi/scheduler.py

```diff
@@ -691,11 +698,11 @@ def dump(self):
 
     @rpc_method()
     def prune(self):
-        logger.info("Starting pruning of task graph")
+        logger.debug("Starting pruning of task graph")
```

Comment:
Ideally these changes would be in a separate commit

Reply:
Will refactor out.